src/Components/calculator.js

```diff
@@ -0,0 +1,46 @@
+function Calculator() {
+  const createDigits = () => {
+    const digits = [];
+
+    // eslint-disable-next-line no-plusplus
```

Comment:
- [ ] Don't disable eslint rules, you can fix the eslint issue you have on the next line so easily, please find a way to fix it instead of disabling the rule, this will help in making your code clean